zerver/lib/tutorial.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+from django.conf import settings
+from zerver.models import get_system_bot, UserProfile
+from zerver.lib.actions import internal_send_private_message
+
+def pm_user_welcome_tips(user):
```

Comment:
This can just go in registration.py, since I don't think any other part of the codebase is going to call it, and it is pretty closely tied to the registration process.
I would also rename it to `send_initial_pms`